pkgs/applications/networking/cluster/k3s/default.nix

```
@@ -137,6 +138,10 @@ let
     nativeBuildInputs = [ git pkg-config ];
```

Is git still required?

```suggestion
    nativeBuildInputs = [ pkg-config ];
```